calenderApp/CalendarController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  CalendarController.swift
+//  calenderApp
+//
+//  Created by Jawaher Alagel on 7/8/20.
+//  Copyright © 2020 Jawaher Alaggl. All rights reserved.
+//
+
+import UIKit
+import FSCalendar
+
+
+
+class CalendarController: UIViewController, FSCalendarDataSource, FSCalendarDelegate {
+    
+    var selectedName: String = "Anonymous"
+    weak var delegate: ViewController!
+    
+    var calendar: FSCalendar!
+    
+    var textContent : String = ""
+    var calnderCallbackObj : CalanderCallback?
+    var currentDateStr : String = ""
+    
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        
+        let height: CGFloat = UIDevice.current.model.hasPrefix("iphone") ? 400 : 300
+        
+        let calendar = FSCalendar(frame: CGRect(x: 0, y: self.navigationController!.navigationBar.frame.maxY, width: view.frame.size.width, height: height))
+        calendar.dataSource = self
+        calendar.delegate = self
+        calendar.allowsMultipleSelection = false
+        view.addSubview(calendar)
+        self.calendar = calendar
+        
+        calendar.calendarHeaderView.backgroundColor = UIColor.lightGray.withAlphaComponent(0)
+        calendar.calendarWeekdayView.backgroundColor = UIColor.lightGray.withAlphaComponent(0)
```

Comment:
make this class property and assign it from there, set the properties from the calling class